_posts/national/united-kingdom/united-kingdom.Rmd

```diff
@@ -62,3 +61,121 @@ summary_path <- file.path("covid-rt-estimates/subnational", folder, "cases/summa
 
 ```{r,  child = system.file("templates/_regional-summary.Rmd", package = "EpiNow2")}
 ```
+
+## Comparison of cases, admissions, and deaths data
+
+### Using Rt to explore transmission in vulnerable populations
+
+We calculate Rt as the average of how many new infections arise from one infected person. But who is the "average" infected person? We only measure a Covid-19 infection when someone gets tested, admitted to hospital, or dies after disease has been confirmed. However, none of these data sources is truly representative of all the infections that have occurred. For example, patients in hospital are more likely to be vulnerable to severe disease (e.g. older or immunocompromised), compared to the average in the general population. This means that each data source represents a slightly different type of "average" person who has been infected.
+
+We can use this feature of each data source to explore how transmission is spreading in different parts of the population. For example, in the UK in August, most new infections were spread between young people, typically less vulnerable to severe disease. This meant the Rt from all test-positive cases rose quicker over that period than the Rt calculated from patients in hospital.
+
+By tracking these comparisons, we can understand how Covid-19 is affecting the most vulnerable populations. To see this more clearly, we plot and compare Rt estimates from each data source.
+
+To explore in more depth how Rt from different data sources can be used to understand transmission dynamics across the population, see our pre-print (currently in peer-review) [here](https://doi.org/10.1101/2020.10.18.20214585).
+
+```{r, echo=FALSE}
+library(magrittr)
```

Comment:
check all deps here are listed in the description and ideally load at the top clearly.